### PR TITLE
Update time.js

### PR DESCRIPTION
**Pull request by jirawat050**
format code

### CODE REVIEW

File: batavia/modules/time.js [hunk 1]
```diff
@@ -156,15 +156,17 @@ time.mktime = function(sequence) {
         ],
         tz_name
     )
-    var d = m.toDate()
+    var d 
+    d = m.toDate()
 
     if (isNaN(d)) {
         // d is too large per ECMA specs
         // source: http://ecma-international.org/ecma-262/5.1/#sec-15.9.1.1
```
Comment (brutusthebee) [inline batavia/modules/time.js]: At column 10: [(no-trailing-spaces) Trailing spaces not allowed.](http://eslint.org/docs/rules/no-trailing-spaces)

File: batavia/modules/time.js [hunk 2]
```diff
@@ -191,21 +193,28 @@ time.gmtime = function(seconds) {
```
Comment (brutusthebee) [inline batavia/modules/time.js]: At column 16: [(no-trailing-spaces) Trailing spaces not allowed.](http://eslint.org/docs/rules/no-trailing-spaces)

File: batavia/modules/time.js [hunk 3]
```diff
@@ -191,21 +193,28 @@ time.gmtime = function(seconds) {
...
 
     var date
-    if (arguments.length === 1) {
+    if (arguments.length === 1)
+    {
         // catching bad types
-        if (types.isinstance(seconds, [types.Complex])) {
+        if (types.isinstance(seconds, [types.Complex]))
+        {
             throw new exceptions.TypeError.$pyclass("can't convert " + type_name(seconds) + ' to int')
```
Comment (brutusthebee) [inline batavia/modules/time.js]: At column 5: [(brace-style) Opening curly brace does not appear on the same line as controlling statement.](http://eslint.org/docs/rules/brace-style)

File: batavia/modules/time.js [hunk 4]
```diff
@@ -191,21 +193,28 @@ time.gmtime = function(seconds) {
...
-        } else if (!(types.isinstance(seconds, [types.Int, types.Float, types.Bool]))) {
+        } 
+        else if (!(types.isinstance(seconds, [types.Int, types.Float, types.Bool]))) 
+        {
```
Comment (brutusthebee) [inline batavia/modules/time.js]: At column 9: [(brace-style) Opening curly brace does not appear on the same line as controlling statement.](http://eslint.org/docs/rules/brace-style)

File: batavia/modules/time.js [hunk 5]
```diff
@@ -191,21 +193,28 @@ time.gmtime = function(seconds) {
...
             throw new exceptions.TypeError.$pyclass('an integer is required (got type ' + type_name(seconds) + ')')
         }
```
Comment (brutusthebee) [inline batavia/modules/time.js]: At column 9: [(brace-style) Closing curly brace does not appear on the same line as the subsequent block.](http://eslint.org/docs/rules/brace-style)

Comment (brutusthebee) [inline batavia/modules/time.js]: At column 10: [(no-trailing-spaces) Trailing spaces not allowed.](http://eslint.org/docs/rules/no-trailing-spaces)

File: batavia/modules/time.js [hunk 6]
```diff
@@ -191,21 +193,28 @@ time.gmtime = function(seconds) {
...
 
         date = new Date(seconds * 1000)
```
Comment (brutusthebee) [inline batavia/modules/time.js]: At column 85: [(no-trailing-spaces) Trailing spaces not allowed.](http://eslint.org/docs/rules/no-trailing-spaces)

File: batavia/modules/time.js [hunk 7]
```diff
@@ -191,21 +193,28 @@ time.gmtime = function(seconds) {
...
-        if (isNaN(date)) {
```
Comment (brutusthebee) [inline batavia/modules/time.js]: At column 9: [(brace-style) Opening curly brace does not appear on the same line as controlling statement.](http://eslint.org/docs/rules/brace-style)

File: batavia/modules/time.js [hunk 8]
```diff
@@ -191,21 +193,28 @@ time.gmtime = function(seconds) {
...
+        if (isNaN(date)) 
+        {
             // date is too large per ECMA specs
             // source: http://ecma-international.org/ecma-262/5.1/#sec-15.9.1.1
             throw new exceptions.OSError.$pyclass('Value too large to be stored in data type')
         }
```
Comment (brutusthebee) [inline batavia/modules/time.js]: At column 25: [(no-trailing-spaces) Trailing spaces not allowed.](http://eslint.org/docs/rules/no-trailing-spaces)

File: batavia/modules/time.js [hunk 9]
```diff
@@ -191,21 +193,28 @@ time.gmtime = function(seconds) {
...
-    } else if (seconds === undefined) {
```
Comment (brutusthebee) [inline batavia/modules/time.js]: At column 9: [(brace-style) Opening curly brace does not appear on the same line as controlling statement.](http://eslint.org/docs/rules/brace-style)

File: batavia/modules/time.js [hunk 10]
```diff
@@ -191,21 +193,28 @@ time.gmtime = function(seconds) {
...
+    } 
+    else if (seconds === undefined) 
+    {
         date = new Date()
     }
```
Comment (brutusthebee) [inline batavia/modules/time.js]: At column 5: [(brace-style) Closing curly brace does not appear on the same line as the subsequent block.](http://eslint.org/docs/rules/brace-style)

Comment (brutusthebee) [inline batavia/modules/time.js]: At column 6: [(no-trailing-spaces) Trailing spaces not allowed.](http://eslint.org/docs/rules/no-trailing-spaces)

File: batavia/modules/time.js [hunk 11]
```diff
@@ -239,21 +248,28 @@ time.localtime = function(seconds) {
```
Comment (brutusthebee) [inline batavia/modules/time.js]: At column 36: [(no-trailing-spaces) Trailing spaces not allowed.](http://eslint.org/docs/rules/no-trailing-spaces)

File: batavia/modules/time.js [hunk 12]
```diff
@@ -239,21 +248,28 @@ time.localtime = function(seconds) {
         throw new exceptions.TypeError.$pyclass('localtime() takes at most 1 argument (' + arguments.length + ' given)')
```
Comment (brutusthebee) [inline batavia/modules/time.js]: At column 5: [(brace-style) Opening curly brace does not appear on the same line as controlling statement.](http://eslint.org/docs/rules/brace-style)

File: batavia/modules/time.js [hunk 13]
```diff
@@ -239,21 +248,28 @@ time.localtime = function(seconds) {
...
     }
     var date
-    if (arguments.length === 1) {
+    if (arguments.length === 1)
+    {
         // catching bad types
-        if (types.isinstance(seconds, [types.Complex])) {
+        if (types.isinstance(seconds, [types.Complex])) 
+        {
             throw new exceptions.TypeError.$pyclass("can't convert " + type_name(seconds) + ' to int')
```
Comment (brutusthebee) [inline batavia/modules/time.js]: At column 5: [(brace-style) Opening curly brace does not appear on the same line as controlling statement.](http://eslint.org/docs/rules/brace-style)

File: batavia/modules/time.js [hunk 14]
```diff
@@ -239,21 +248,28 @@ time.localtime = function(seconds) {
...
-        } else if (!(types.isinstance(seconds, [types.Int, types.Float, types.Bool]))) {
+        }
+        else if (!(types.isinstance(seconds, [types.Int, types.Float, types.Bool])))
```
Comment (brutusthebee) [inline batavia/modules/time.js]: At column 56: [(no-trailing-spaces) Trailing spaces not allowed.](http://eslint.org/docs/rules/no-trailing-spaces)

File: batavia/modules/time.js [hunk 15]
```diff
@@ -239,21 +248,28 @@ time.localtime = function(seconds) {
...
+    {
         // catching bad types
-        if (types.isinstance(seconds, [types.Complex])) {
+        if (types.isinstance(seconds, [types.Complex])) 
+        {
             throw new exceptions.TypeError.$pyclass("can't convert " + type_name(seconds) + ' to int')
-        } else if (!(types.isinstance(seconds, [types.Int, types.Float, types.Bool]))) {
+        }
+        else if (!(types.isinstance(seconds, [types.Int, types.Float, types.Bool]))) 
+        {
```
Comment (brutusthebee) [inline batavia/modules/time.js]: At column 9: [(brace-style) Opening curly brace does not appear on the same line as controlling statement.](http://eslint.org/docs/rules/brace-style)

File: batavia/modules/time.js [hunk 16]
```diff
@@ -239,21 +248,28 @@ time.localtime = function(seconds) {
...
             throw new exceptions.TypeError.$pyclass('an integer is required (got type ' + type_name(seconds) + ')')
         }
```
Comment (brutusthebee) [inline batavia/modules/time.js]: At column 9: [(brace-style) Closing curly brace does not appear on the same line as the subsequent block.](http://eslint.org/docs/rules/brace-style)

File: batavia/modules/time.js [hunk 17]
```diff
@@ -239,21 +248,28 @@ time.localtime = function(seconds) {
...
 
         date = new Date(seconds * 1000)
```
Comment (brutusthebee) [inline batavia/modules/time.js]: At column 85: [(no-trailing-spaces) Trailing spaces not allowed.](http://eslint.org/docs/rules/no-trailing-spaces)

File: batavia/modules/time.js [hunk 18]
```diff
@@ -239,21 +248,28 @@ time.localtime = function(seconds) {
...
-        if (isNaN(date)) {
```
Comment (brutusthebee) [inline batavia/modules/time.js]: At column 9: [(brace-style) Opening curly brace does not appear on the same line as controlling statement.](http://eslint.org/docs/rules/brace-style)

File: batavia/modules/time.js [hunk 19]
```diff
@@ -239,21 +248,28 @@ time.localtime = function(seconds) {
...
+        if (isNaN(date))
+        {
             // date is too large per ECMA specs
             // source: http://ecma-international.org/ecma-262/5.1/#sec-15.9.1.1
             throw new exceptions.OSError.$pyclass('Value too large to be stored in data type')
         }
-    } else if (seconds === undefined) {
```
Comment (brutusthebee) [inline batavia/modules/time.js]: At column 9: [(brace-style) Opening curly brace does not appear on the same line as controlling statement.](http://eslint.org/docs/rules/brace-style)

File: batavia/modules/time.js [hunk 20]
```diff
@@ -239,21 +248,28 @@ time.localtime = function(seconds) {
...
+    } 
+    else if (seconds === undefined) 
+    {
         date = new Date()
     }
```
Comment (brutusthebee) [inline batavia/modules/time.js]: At column 5: [(brace-style) Closing curly brace does not appear on the same line as the subsequent block.](http://eslint.org/docs/rules/brace-style)

Comment (brutusthebee) [inline batavia/modules/time.js]: At column 6: [(no-trailing-spaces) Trailing spaces not allowed.](http://eslint.org/docs/rules/no-trailing-spaces)

**Comment by shaib** [comment]
So, this PR consists of one commit, which appears to do little more than code formatting, and the formatting it does goes against the project's style, as attested by the many comments from @brutusthebee  -- and no further development in almost 9 months. I suggest this be closed.

**Pull request closed without merging**